src/main/java/duck/MainWindow.java

```diff
@@ -0,0 +1,62 @@
+package duck;
+
+import java.io.IOException;
+
+import duck.exception.UnknownCommandException;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
+ * Controller for the main GUI.
+ */
```

Comment:
Should there be a new line break between imports and javadoc?

```suggestion
import javafx.scene.layout.VBox;

/**
 * Controller for the main GUI.
 */
```